vendor/modules.txt

```diff
@@ -1,10 +1,12 @@
 # github.com/appscode/jsonpatch v0.0.0-20190108182946-7c0e3b262f30
+## explicit
```

Comment:
question: why some modules are marked as `explicit` with version upgrade?

Reply:
Section about vendoring explains that: https://golang.org/doc/go1.14